frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -293,6 +304,14 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                     : 'none'
             },
         ],
+        isNavShown: [
+            (s) => [s.isNavShownMobile, s.mobileLayout],
+            (isNavShownMobile, mobileLayout): boolean => !mobileLayout || isNavShownMobile,
+        ],
+        isNavCollapsed: [
+            (s) => [s.isNavCollapsedDesktop, s.mobileLayout],
+            (isNavCollapsedDesktop, mobileLayout): boolean => !mobileLayout && isNavCollapsedDesktop,
+        ],
```

Comment:
It's not clear to me why these two selectors which seem to be opposites of each other rely on different parameters (one on mobile and the other on desktop). Played around with the branch and everything worked really well so I'm sure it's intended

Reply:
They're quite different: "collapsed" - nav is icon-only, "shown" - nav is, well, shown